perf_test/README.md

```diff
@@ -16,7 +16,8 @@ The core design principle beneath Howitzer lies in "exploding" K6 parameter iter
 When Howitzer is run, the bash script will create directories `render`, `results` and `summary`:
 - `render` stores the "exploded" k6 tests using the k6 template specified in the test config for each k6_opts combination.
 - Then the script will run each individual k6 test in this `render` directory, storing the human readable results as text in `results` directory.
-- It will also store the corresponding json file of the result into `summary` directory that we can later use for MLFlow.
+- It will also store the corresponding json file of the result into `summary` directory.
+- The scraper script then consolidates the results from the K6 test, the input test conditions etc. into a single `summary.json` file for the tests that were run.
```

Comment:
I think since now we have multiple scripts, let's be explicit and say the `perf_test.scripts.scraper` script, instead of just scraper script.